docs/operator-guides/configuring-connector-resources.md

```diff
@@ -33,6 +53,28 @@ Instance-wide requirements are the simplest requirement to configure. All that i
 
 ## Configuring Connector-Specific Requirements
 
+You can configure resource allocation through the "Connector resource allocation" dropdown in the Settings tab for each connector. You can also configure this with the API.
```

Comment:
The UI and the configuration itself of actor-level resource requirements are only available in SME.  Worth noting because this page says OSS and SME at the top!